crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -3710,30 +3704,12 @@
     "",
     "BAD_OPCODE"
   ],
-  [
-    "1",
```

Comment:
Can you keep test, and change result and description?

Reply:
commit 955424cd4c1dd72f4c1028b3bcdd3dd017bb6309